registry.go

```diff
@@ -71,15 +71,15 @@ func Logout(ctx context.Context, store Store, registryName string) error {
 // Credential returns a Credential() function that can be used by auth.Client.
 func Credential(store Store) func(context.Context, string) (auth.Credential, error) {
 	return func(ctx context.Context, reg string) (auth.Credential, error) {
-		reg = mapAuthenticationRegistryName(reg)
+		reg = MapRegistryNameRegistry1DockerIo(reg)
 		if reg == "" {
 			return auth.EmptyCredential, nil
 		}
 		return store.Get(ctx, reg)
 	}
 }
 
-func mapStoreRegistryName(registry string) string {
+func MapRegistryNameDockerIo(registry string) string {
```

Comment:
The method name is not meaningful.

Reply:
This method maps a registry name to a key for credential store. Therefore, `func ServerAddressFromRegistry(registry string) string` with documentation is preferable.

Reply:
changed accordingly.